js/apps/admin-ui/src/user/UserIdentityProviderLinks.tsx

```diff
@@ -87,7 +87,7 @@ export const UserIdentityProviderLinks = ({
       (x) => x.identityProvider,
     );
 
-    return getAvailableIdPs()?.filter(
+    return (await getAvailableIdPs())?.filter(
```

Comment:
I think we can avoid querying all brokers and have a specific endpoint to return the brokers the user is not yet linked. I can do now or as a follow-up.